docs/guides/operator-guide-aws-marketplace.md

```diff
@@ -350,6 +350,23 @@ The following table includes some additional commands that might help you manage
 | Runs one iteration of `logrotate` manually, without changing the scheduled interval. | `sudo logrotate -f /etc/logrotate.conf --force` |
 | Reloads `syslog-ng`. | `sudo /usr/sbin/syslog-ng-ctl reload` |
 
+### UID2 Operator error codes
+
+The following table lists possible errors while setting up private operator.
+
+:::note
+This applies only to Private Operators starting from Q2 2024.
+:::
+
+| Error Code | Issue | Steps to Resolve |
+| :--- | :--- | :--- |
+| AW01 | MissingInstanceProfile |  Attach an IAM instance profile to the EC2 instance with the required permissions. The UID2 Operator needs these permissions to access configurations from AWS Secrets Manager. |
```

Comment:
1) If the issue is an actual code value that's returned, this is fine.
If not, it should be in separate words: Missing instance profile, Configuration not found, etc.

2) If the permissions associated with the profile or the instance? If it's the instance, this is fine. But I think probably it's the profile. If so, it should say this:

Attach an IAM instance profile with the required permissions to the EC2 instance.

Reply:
It is code that is returned. And it is an instance